.github/workflows/schedule.py

```diff
@@ -38,11 +38,23 @@ class FrontMatter(TypedDict, total=False):
     type: str
     labels: list[str]
     assignees: list[str]
+    _project_item_priority: str
     _repository: str
     _start: str
     _period: str
 
 
+azul_project_number = '3'
+azul_project_id = 'PVT_kwDOAfSQ384BCJY8'
```

Comment:
I don't remember agreeing that it would be OK to hard-code all this

```
$ gh project list --format json --owner DataBiosphere --jq '.projects[]|select(.title == "Azul")|.number'
3
$ gh project field-list 3 --owner=DataBiosphere --format=json --jq '.fields[]|select(.name == "Priority")|.options'
[
  {
    "id": "bdf99630",
    "name": "++"
  },
  {
    "id": "3333032d",
    "name": "+"
  },
  {
    "id": "5e2e7b47",
    "name": "-"
  },
  {
    "id": "b903d2dc",
    "name": "--"
  }
]
```

You can probably extract the owner from the environment. The only thing you'd have to hard-code is the project and field name.